RecastDemo/Source/Sample.cpp

```diff
@@ -37,23 +37,37 @@
 
 unsigned int SampleDebugDraw::areaToCol(unsigned int area)
 {
-	switch(area)
+	unsigned int col;
+
+	unsigned char ceil = (area & SAMPLE_POLYAREA_CEIL_MASK);
+	switch(ceil)
 	{
-	// Ground (0) : light blue
-	case SAMPLE_POLYAREA_GROUND: return duRGBA(0, 192, 255, 255);
+	// No ceil, white
+	case 0: col = duRGBA(255, 255, 255, 255); break;
```

Comment:
Actually 0 is a value that should not appear anymore in the sample, due to chosen area type and flags values.